openbook/urls.py

```diff
@@ -110,6 +109,7 @@
     path('block/', BlockUser.as_view(), name='block-user'),
     path('unblock/', UnblockUser.as_view(), name='unblock-user'),
     path('report/', ReportUser.as_view(), name='report-user'),
+    path('subscribe/notifications/', SubscribeUserNotifications.as_view(), name='subscribe-user-notifications'),
```

Comment:
notifications/subscribe and `SubscribeToUserNotifications` for the APIView please 🙌